src/containers/Dashboard/DashboardContainer.tsx

```diff
@@ -31,7 +32,13 @@ export const DashboardContainer = (): ReactElement => {
 
     return (
         <div className="dashboard-container">
-            <TopBar />
+            <TopBar
+                logOut={() => {
```

Comment:
Move this logic to a redux function. Think of it as layers where in redux you do all the data management, React components are here just to display it and call actions.